src/client/store/data/hooks/useTableData.ts

```diff
@@ -8,19 +8,19 @@ import { useCountryIso } from '@client/hooks'
 
 import { useOriginalDataPointData } from './useOriginalDataPointData'
 
-export const useTableData = (props: { table: Table }): TableData => {
+export const useTableData = (props: { table: Table }): RecordAssessmentData => {
   const { table } = props
   const countryIso = useCountryIso()
   const { odp } = table.props
   const assessment = useAssessment()
   const cycle = useCycle()
   const country = useAssessmentCountry()
 
-  const tableData = useAppSelector((state) => state.data[assessment.props.name][cycle.name].tableData)
+  const tableData = useAppSelector((state) => state.data.tableData)
   const odpData = useOriginalDataPointData() ?? {}
   const showOriginalDatapoints = useShowOriginalDatapoints()
 
-  if (!tableData?.[countryIso]) return {} as TableData
+  if (!tableData?.[assessment.props.name]?.[cycle.name]?.[countryIso]) return {}
```

Comment:
better to add  a util method  to extract tableData?.[assessment.props.name]?.[cycle.name]?.[countryIso] ? what do you think ?